codespell_lib/data/dictionary.txt

```diff
@@ -25094,6 +25094,7 @@ stablility->stability
 stablilization->stabilization
 stablize->stabilize
 stach->stack
+stadnard->standard
```

Comment:
Can you add the plural and standardised, standardising and standardisation please.

Reply:
Done